### PR TITLE
README: Hide deprecated Celery configuration environment variables

### PR DESCRIPTION
**Pull request by JayH5**
[Rendered](https://github.com/praekeltfoundation/docker-django-bootstrap/blob/readme-hide-deprecated-opts/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,23 +161,37 @@ The following environment variables can be used to configure Celery, but, other
...
-* Required: no
-* Default: none
-* Celery option: `-l`/`--loglevel`
-
-#### `CELERY_CONCURRENCY`:
-Note that by default Celery runs as many worker processes as there are processors. **We instead default to 1 worker process** here to ensure containers use a consistent and small amount of resources. If you need to run many worker processes, they should be in separate containers.
-* Required: no
-* Default: **1**
-* Celery option: `-c`/`--concurrency`
+<details>
```
Comment (JayH5) [inline README.md:164]: `<details>` is not natively supported by Markdown so we've gotta use inline HTML.

Comment (miltontony) [reply README.md:164]: Interesting.. the preview of the file suggests it'll be displayed collapsed. Which is quite cool!

**Review by miltontony — APPROVED** [review]
👍 🍰

File: README.md (hunk 1 above)
Comment (JayH5) [reply README.md:164]: Found it here: https://github.com/dear-github/dear-github/issues/166

**Pull request merged**